Autocoders/Python/src/fprime_ac/utils/version.py

```diff
@@ -1,4 +1,5 @@
 """ fprime version handling and reporting """
```

Comment:
Should we take this opportunity to change this path?  We will (hopefully) blow away the AC package soon, so this file will need to find somewhere else to live.

Reply:
Later.

Reply:
I considered it but also noticed that some other part of the `Autocoders/` package use that module. Didn't want to break those dependencies or duplicate the code so I left it there. I'm ok breaking things if that's what we want to do.